greenplumpython/table.py

```diff
@@ -101,7 +101,7 @@ def fetch(self, all: bool = True) -> Iterable:
         return self._db.execute(self._build_full_query())
 
     def save_as(
-        self, table_name: str, temp: bool = True, column_names: Iterable[str] = []
+        self, table_name: str, temp: bool = False, column_names: Iterable[str] = []
```

Comment:
Why this default value is changed?

Reply:
Because I realized that if I want to save some data to the database, I would expect that the data is persistent after my work is finished. 

But if I want to create a function, I would expect that the function is deleted as soon as I finish using it.

Reply:
Interesting!